### DISCUSSION TITLE
Express.js app not serving static files

### DISCUSSION THREAD

**Discussion by Oren-Lindsey** [vercel/vercel#7572] [Help]
My express.js app (deployed on vercel obviously) isn't serving static files. I think I know why: I set "public" to be the static file directory in the gui but I keep getting this warning: 
`Warning: Due to 'builds' existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings
`
 so I think that it's overriding that setting. But I don't know how to fix it. Also my app is pure express, no framework. If anyone can help I'd be very grateful

My code is [here](https://github.com/Oren-Lindsey/random-art)
Edit: I'm not sure if that is the problem, but I think it is. If you find some other problem please let me know about it

**Comment by raldincasidar-studio** [comment]
Im having the same problem, I wish someone could help us

**Comment by ubyjerome** [comment]
Same problem here for like 2 months
